ocs_ci/utility/utils.py

```diff
@@ -446,6 +446,8 @@ def mask_secrets(plaintext, secrets):
                 plaintext = [string.replace(secret, "*" * 5) for string in plaintext]
             else:
                 plaintext = plaintext.replace(secret, "*" * 5)
+    if secrets == ["*"]:
+        plaintext = "*" * 10
```

Comment:
What is the purpose of this change?

Reply:
I wanted to mask all the CR with IPs (IpAddressPool). I will probably remove this.